backend/battling/forms.py

```diff
@@ -15,15 +20,25 @@ def __init__(self, *args, **kwargs):
         self.fields["opponent"].queryset = User.objects.exclude(id=self.initial["creator_id"])
 
 
+POSITION_CHOICES = [(1, 1), (2, 2), (3, 3)]
+
+
 class CreateTeamForm(forms.ModelForm):
     class Meta:
         model = Team
         fields = [
             "pokemon_1",
+            "pokemon_1_position",
             "pokemon_2",
+            "pokemon_2_position",
             "pokemon_3",
+            "pokemon_3_position",
         ]
 
+    pokemon_1_position = forms.ChoiceField(choices=POSITION_CHOICES, label="Pokemon position")
```

Comment:
I think you should use TypedChoiceField to coerce the value to int. See: https://stackoverflow.com/questions/3673833/typedchoicefield-or-choicefield-in-django